src/libs/elektra/kdb.c

```diff
@@ -456,7 +458,7 @@ static int elektraGetCheckUpdateNeeded (Split * split, Key * parentKey)
 
 		if (backend->getplugins[RESOLVER_PLUGIN])
 		{
-			// GLOBAL: getresolver
+			// GLOBAL: getresolver [foreach]
```

Comment:
I think we missed at the meeting that for `[foreach]` a pre/post might be useful. (Post execute after line 465)